### DISCUSSION TITLE
Copy paste from excel to nocodb [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1919] [Discord Threads]
> 🙋‍♂️ **Eldynn**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Copy/paste from excel to nocodb is in feature list   
####    
#### Importing excel to nocodb - should be soon available. This was avaialble in <0.90 - as there has been refactoring to our api structure - it is temporarily commented out.   
####    
#### PG jsonb field : do let us know after trying. We should allow creating it. And we do show JSON fields in monaco editor within a cell I think  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966214761429745664)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
